simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -49,3 +50,84 @@ export const ExtraInfo = () => (
     </div>
   </div>
 );
+
+export const Footer = () => (
+  <div className={styles.footer}>
+    <div className={styles['main-card-container']}>
+      <div className={styles.card}>
+        <div className={styles['card-header']}>
+          <div className={styles.simpleQ_image_container}>
+            <img
+              src="/images/new_logo.svg"
+              className={styles['new-logo-image']}
+              alt="simplQ new logo"
+            />
+          </div>
+          <div className={styles.logo_header_container}>
+            <h1>SimplQ</h1>
+          </div>
+        </div>
+        <div className={styles['card-body']}>
+          <div className={styles['description-content-container']}>
+            <b className={styles['description-content']}>
+              SimplQ is a completely web based queue management solution that anyone can use to
+              create instant virtual queues.
+            </b>
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.card}>
+        <div className={styles['card-header']}>
+          <div className={styles['center-item']}>
+            <h1>Open source</h1>
+          </div>
+        </div>
+        <div className={styles['card-body']}>
+          <div className={styles['open-source-content-container']}>
+            <p>
+              <b>
+                <span>SimplQ </span>
+                is open source. Be part of the
+                <span> SimpleQ </span>
+                community.
+              </b>
+            </p>
+            <ForkOnGithubButton />
+          </div>
+        </div>
+      </div>
+
+      <div className={styles.card}>
+        <div className={styles['card-header']}>
+          <div className={styles['center-item']}>
+            <h1>Keep in touch</h1>
+          </div>
+        </div>
```

Comment:
Could you try to make the headers simplq, open source, keep in touch line up exactly vertically with the content below? Maybe using a table would work.

Reply:
Hey @maaverik , I will try 👍

Reply:
I couldn't give it time to this PR, sorry guys. Will fix it.

Reply:
No worries. Work at your own pace. In case you're not going to continue on a task, just make sure you mention it in a comment or unassign yourself.

Reply:
And now that I think about it, a table would make the alignments easier, but the responsiveness a flexbox provides would be lost, so scrap the table idea.

Reply:
hey @maaverik ,can you review it again, please?